plugins/telink-dfu/fu-telink-dfu-hid-device.c

```diff
@@ -0,0 +1,370 @@
+/*
+ * Copyright 2024 Mike Chang <Mike.chang@telink-semi.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1-or-later
+ */
+
+#include "config.h"
+
+#ifdef HAVE_HIDRAW_H
+#include <linux/hidraw.h>
+#include <linux/input.h>
+#endif
+
+#include "fu-telink-dfu-archive.h"
+#include "fu-telink-dfu-hid-device.h"
+#include "fu-telink-dfu-struct.h"
+
+struct _FuTelinkDfuHidDevice {
+	FuHidDevice parent_instance;;
+};
+
+G_DEFINE_TYPE(FuTelinkDfuHidDevice, fu_telink_dfu_hid_device, FU_TYPE_HID_DEVICE)
+
+#define FU_TELINK_DFU_HID_DEVICE_SHOW_EDP_INFO	0
```

Comment:
I don't think we want any of the FU_TELINK_DFU_HID_DEVICE_SHOW_EDP_INFO stuff; it's all in the descriptor parsing, no?

Reply:
resolved in commit 678e510a9c1d03ab044f96b1a4716488e838ecf3